SQL_module/SubjectForm_Task2/electives.php

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
+    <title> Form task </title>
+
+</head>
+
+<body>
+
+    <?php
+
+        include "./validation.php";  
+
+        $url = $_SERVER['REQUEST_URI'];
+
+        $dirname = dirname($url);
+        $id = basename($url);
+        $pattern_basename = '/^[1-9][0-9]*$/';
+
+        //To prevent requests like /electives.php/1/2/3
+        if (!preg_match($pattern_basename, $id) || preg_match($pattern_basename, basename($dirname))){           
+            die("Incorrect URL.");
+        }
+
+        if(!checkIfIDExist($id)){
+            die("Incorrect ID!");
+        }
+
+        $requestMethod = $_SERVER['REQUEST_METHOD'];
+        if ($requestMethod == "GET") {
+            printSubject($id);
+        } 
+
+        if ($requestMethod == "POST") {
+            
+            validateField('updated_title', MAX_LENGTH_TITLE, $valid, $errors);
```

Comment:
I think you can extract all these literals for the name of the field as constants.